tests/acceptance/helpers/httpHelper.js

```diff
@@ -49,3 +59,201 @@ exports.checkOCSStatus = function (response, message) {
     throw Error(message + ' Status:' + statusCode)
   }
 }
+
+/**
+ *
+ * @param {string} url
+ * @param {object} params
+ * @param {string} userId
+ * @param {object} header
+ *
+ * @returns {node-fetch}
+ */
+exports.requestWebdavEndpoint = function (url, params, userId = 'admin', header = {}) {
+  const headers = { ...this.createAuthHeader(userId), ...header }
+  const options = { ...params, headers }
+  return fetch(url, options)
```

Comment:
```suggestion
const  response =  await fetch(url, options)
const jsonResponse = await response.json()
return jsonResponse
```
What do you think???

Reply:
We can do that but in many scenarios, we don't even need to parse the response.